src/lib/support/DefaultStorageKeyAllocator.h

```diff
@@ -198,16 +198,18 @@ class DefaultStorageKeyAllocator
     }
     static StorageKeyName SubscriptionResumptionMaxCount() { return StorageKeyName::Formatted("g/sum"); }
 
-    // Scene Count in an Endpoint's SceneTable
+    // Number of scenes stored in a given endpoint's scene table, across all fabrics.
     static StorageKeyName EndpointSceneCountKey(EndpointId endpoint) { return StorageKeyName::Formatted("g/scc/e/%x", endpoint); }
 
-    // General data of all scenes ID belonging to a Fabric on a specific endpoint
+    // Stores the scene count for a fabric and a map between scene storage ids <sceneId, groupId> and sceneIndex for a specific
+    // Fabric
     static StorageKeyName FabricSceneDataKey(FabricIndex fabric, EndpointId endpoint)
     {
         return StorageKeyName::Formatted("f/%x/e/%x/sc", fabric, endpoint);
     }
 
-    // Scene key, Fabric scoped
+    // Scene key, Fabric scoped, the value of the idx is held in FabricSceneData in a map between <scene_id, group_id> and
```

Comment:
```suggestion
    // Stores the actual scene data for a given scene on a given endpoint for a particular fabric. 
    // idx corresponds to the indices read from FabricSceneDataKey.
```
or so, if I understand this correctly now.